client/css/ImageGallery.scss

```diff
@@ -45,57 +43,44 @@
 
 
 .arrowsThumbnailContainer {
-  margin: 0 1%;
+  align-items: center;
   display: flex;
   flex-direction: column;
-  order: 1;
-  width: 15%;
-  align-items: center;
-  justify-content: space-between;
+  height: 100%;
 }
 
 .thumbnailContainer {
   display: flex;
   flex-direction: column;
-  width: 100%;
   align-items: center;
-  justify-content: center;
-}
-
-.thumbnail {
-  height: 100px;
-  width: 100px;
-  margin: 5% 0;
-  background-size: 100%;
-  background-position: center;
-  background-repeat: no-repeat;
 }
 
 .thumbnailInput {
-  height: 100px;
-  width: 100px;
-  margin: 5px;
-  padding: 0;
-  border: 0;
-  background-size: 100%;
+  border: 2px solid #777;
+  border-radius: 3px;
+  background-size: cover;
   background-position: center;
   background-repeat: no-repeat;
+  height: 60px;
+  margin: 10px 0;
+  transition: border-color 200ms;
+  width: 60px;
+
+  &:hover {
+    border-color: goldenrod;
+    transition: border-color 200ms;
+  }
 }
 
+.upArrow,
+.downArrow,
+.rightArrow,
 .leftArrow {
-  left: 102px;
   cursor: pointer;
-}
-
-.rightArrow {
-  cursor: pointer;
-}
+  transition: color 200ms;
 
-#selected {
```

Comment:
I like the change in color on hover but I think we need to keep the selected id as well. The thumbnail of the current image being displayed should have a constant highlighted border around it.

Reply:
Good call. Plus if we put it back, then we can uncomment out those tests 😄 .